x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/google_workspace_mfa_enforcement_disabled.json

```diff
@@ -33,5 +33,5 @@
   ],
   "timestamp_override": "event.ingested",
   "type": "query",
-  "version": 9
+  "version": 11
```

Comment:
This was already bumped from `7` to `9` in the last `8.2 Rules` PR and there were no other content changes: https://github.com/elastic/kibana/pull/129526/files#diff-f6a6a4e4fa25f5387375c98a480cb4c9634cada7e549f1df99a70404b5fd4a13R36

Reply:
Double bump is the result of manual bumps done from the following.

https://github.com/elastic/detection-rules/pull/1870/files